src/pages/Autolending/GenderRadios.vue

```diff
@@ -0,0 +1,126 @@
+<template v-model="isEnabled">
+	<div class="gender-radios">
+		<template v-if="isEnabled">
+			<kv-radio
+				label-set="genderRadioSet"
+				name-set="genderRadio"
+				value="both"
+				v-model="gender"
+				class="both-radio"
+			>
+				Both
+			</kv-radio>
+			<br>
+			<kv-radio
+				label-set="genderRadioSet"
```

Comment:
Since this is used for the input id, this value needs to be different for each radio button (and also unique on the entire page, so maybe something like `genderRadioSetMale`)